rosperch/scripts/wip/mission_commands.py

```diff
@@ -1,17 +1,83 @@
 #!/usr/bin/env python3
-# Based on the simple publisher/subscriber tutorial on the ROS tutorial page:
-# https://github/ros/ros_tutorials.com
+# Decription:
+# This program takes user input commands and sends them
+# to a seperate program that deals with making the motors
+# on the ROSPerch move.
+# Adapted Sources:
+# Motor command code in auto_driver.py is adapted 
+# from the UTAP 2020 code at 
+# https://github.com/jdicecco/UTAP/
+# License:
 # Software License Agreement (GPLv3 License)
+# Find the full agreement at https://github.com/amichael1227/ROSPerch/blob/master/LICENSE
 
+
+# Imports the necessary libraries and messages
 import threading
 import rospy
 import time
 from std_msgs.msg import Bool
 from rosperch.msg import Commands
 
-systemready = True #Assume it's ready, if it isn't it'll tell you
+# Sets up system state varible
+systemready = True # Assume it's ready, if it isn't it'll tell you
+
+# Talker node function
+def talker():
+    global systemready
+    pub = rospy.Publisher('motorcommands', Commands, queue_size=10) # Publish to motorcommands topic
+    rate = rospy.Rate(10) # Sets the ROS rate to 10hz
+    while not rospy.is_shutdown(): # Runs the code only while the roscore is running
+        print("Mission Commands:")
+        print("Drive, Right Turn, Left Turn, Stop")   
+        command_input = input("Please Input a Command: ") # Ask user for mission command
+        print()
+
+        # What to do if the user sends the Drive Command
+        if command_input.upper() == 'DRIVE':
+            mission_parameter = float(input("Distance in Meters: ")) # Asks user for Distance
```

Comment:
Make drive send as all caps

Reply:
Fixed issue and set the command_input variable for drive to all caps.